pkgs/data/icons/vimix-cursors/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv, fetchFromGitHub, fetchpatch, xcursorgen, python3Packages, bash }:
+
+stdenv.mkDerivation rec {
+  pname = "vimix-cursors";
+  version = "2020-02-24";
+
+  src = fetchFromGitHub {
+    owner = "vinceliuice";
+    repo = "Vimix-cursors";
+    rev = version;
+    sha256 = "0bij0x916lbxph28i3d4hz5l6mzk5fa68mf21jnl7y9rpxx07xsd";
+  };
+
+  nativeBuildInputs = [ python3Packages.cairosvg xcursorgen ];
+
+  patches = [
+    (fetchpatch {
```

Comment:
What is the function of these patches? It might be a good idea to add a comment above each one (https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-patch-comment.md)